globus_automate_client/action_client.py

```diff
@@ -44,11 +42,7 @@ def introspect(self, **kwargs) -> GlobusHTTPResponse:
         such as its expected ``action_scope``, its ``input_schema``, and who to
         contact when there's trouble.
         """
-        headers: Dict = {}
-        if self.authorizer is not None:
-            self.authorizer.set_authorization_header(headers)
-        resp = requests.get(self.base_url, headers=headers)
-        return self.default_response_class(resp, client=self)
+        return self.get("")
```

Comment:
Also a good sign that this is gone. Should it be `self.get("/")`. It kinda reads better, but probably doesn't matter or perhaps even would cause a problem?

Reply:
Ideally it wouldn't matter but I noticed the introspection endpoint in APT doesn't have a trailing "/" so making it `self.get("/")` results in 404s (which also means that introspecting an action with action-url's ending in "/" throws 404). I like the `self.get("/")` and will probably make the endpoint agnostic to trailing slashes.

Reply:
Nevermind, it looks like this needs to be self.get("") to work with the Flows service as it exists. The best fix for this would be making our endpoints agnostic to trailing slashes. if we only do it in the CLI we have a lot of updates to do in docs/examples/code.

Reply:
Agreed, we should plan to make that change. Should be as easy as setting the property on the blueprint about trailing slashes.